tests/quality_pragmaitc/test_refinement.py

```diff
@@ -363,26 +355,31 @@ def testGetApplicableQC():
     interp.addQualityConstraint(qc)
 
     fullContext.append(context)
-    # assertEquals(null, goal.isAchievable(fullContext, interp))
 
-    assert qc in goal.getInterpretation().getQualityConstraints(
+    assert stricter not in goal.getInterpretation().getQualityConstraints(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4895731)